nodejs/lancedb/table.ts

```diff
@@ -603,29 +602,24 @@ export class LocalTable extends Table {
       });
     }
 
-    // The query type is auto or vector
-    // fall back to full text search if no embedding functions are defined and the query is a string
-    if (queryType === "auto" && getRegistry().length() === 0) {
-      return this.query().fullTextSearch(query, {
-        columns: ftsColumns,
-      });
+    // TODO: Support multiple embedding functions
+    const embeddingFunc: EmbeddingFunctionConfig | undefined =
+      this.getEmbeddingFunctions().values().next().value;
+    if (!embeddingFunc) {
+      // The query type is auto and string
+      // fall back to full text search if no embedding functions are defined and the query is a string
+      if (queryType === "auto" && typeof query === "string") {
+        return this.query().fullTextSearch(query, {
+          columns: ftsColumns,
+        });
+      }
+      return this.query().nearestTo(
+        Promise.reject(
+          new Error("No embedding functions are defined in the table"),
+        ),
+      );
```

Comment:
It looks like you are passing a failed promise to the `nearestTo` function.  Is this intentional?

Reply:
I think I did this for passing some tests, just fixed it